components/apim-analytics/org.wso2.analytics.solutions.apim.analytics/widgets/APIMApiBackendUsageSummary/src/APIMApiBackendUsage.jsx

```diff
@@ -0,0 +1,171 @@
+            width: '90%',
+            height: '10%',
+            margin: 'auto',
+        },
+        form: {
+            display: 'flex',
+            flexWrap: 'wrap',
+        },
+        formControl: {
+            margin: '5%',
+            minWidth: 120,
+        },
+        textField: {
+            margin: '5%',
+            minWidth: 120,
+        },
+        selectEmpty: {
+            marginTop: 10,
+        },
+    };
+    if (usageData.length === 0) {
+        return (
+            <div style={styles.paperWrapper}>
+                <Paper
+                    elevation={1}
+                    style={styles.paper}
+                >
+                    <Typography variant='h5' component='h3'>
+                        <FormattedMessage id='nodata.error.heading' defaultMessage='No Data Available !' />
+                    </Typography>
+                    <Typography component='p'>
+                        <FormattedMessage
+                            id='nodata.error.body'
+                            defaultMessage='No data available for the selected options.'
+                        />
+                    </Typography>
+                </Paper>
+            </div>
+        );
+    }
+    return (
+        <Scrollbars
+            style={{ height }}
+        >
+            <div style={{
+                padding: '5% 5%',
+            }}
+            >
+                <div style={styles.headingWrapper}>
+                    <h3 style={{
+                        borderBottom: themeName === 'dark' ? '1px solid #fff' : '1px solid #02212f',
+                        paddingBottom: '10px',
+                        margin: 'auto',
+                        marginTop: 0,
+                        textAlign: 'left',
+                        fontWeight: 'normal',
+                        letterSpacing: 1.5,
+                    }}
+                    >
+                        <FormattedMessage id='widget.heading' defaultMessage='API BACKEND USAGE SUMMARY' />
+                    </h3>
+                </div>
+                <div style={styles.formWrapper}>
+                    <form style={styles.form}>
+                        <FormControl style={styles.formControl}>
+                            <InputLabel shrink htmlFor='api-createdBy-label-placeholder'>
+                                <FormattedMessage id='createdBy.label' defaultMessage='API Created By' />
+                            </InputLabel>
+                            <Select
+                                value={apiCreatedBy}
+                                onChange={apiCreatedHandleChange}
+                                input={<Input name='apiCreatedBy' id='api-createdBy-label-placeholder' />}
+                                displayEmpty
+                                name='apiCreatedBy'
+                                style={styles.selectEmpty}
+                            >
+                                <MenuItem value='All'>All</MenuItem>
```

Comment:
Shouldn't 'me' and 'all' be internationalized?

Reply:
Fixed in commit 2ab780a